lectures/long_run_growth.md

```diff
@@ -11,42 +11,57 @@ kernelspec:
+Looking at his graph and how it set the geopolitical stage for "the American (20th) century" naturally 
+tempts one to want a counterpart to his graph for 2014 or later.
 
-This lecture looks at different growth trajectories across countries over the long term. 
+As we'll see, reasoning just by analogy, this graph perhaps set the stage for an "XXX (21st) century", where you get to fill in a country for our XXX.
 
-While some countries have experienced long term rapid growth across that has last a hundred years, others have not. 
+As we gather data to construct those two graphs, we'll also study growth experiences for a number of countries for time horizons extending as far back as possible.
 
-First let us import the packages needed to explore what the data says about long run growth.
+These graphs will portray how the "Industrial Revolution" began in Britain in the late 18th century, then migrated to one country after another.  
+
+In a nutshell, this  lecture records  growth trajectories of various  countries over long time periods. 
+
+While some countries have experienced long term rapid growth across that has lasted a hundred years, others have not. 
+
+Since populations differ across country and within a country vary over time, it will
+be interesting to describe both total GNP and GNP per capita as it evolves within a country.
+
+First let's import the packages needed to explore what the data says about long run growth
 
 ```{code-cell} ipython3
 import pandas as pd
 import os
 import matplotlib as mpl
 import matplotlib.pyplot as plt
+import matplotlib.cm as cm
 import numpy as np
+from collections import namedtuple
 from matplotlib.lines import Line2D
 ```
 
++++ {"user_expressions": []}
+
+## Setting up
+
++++ {"user_expressions": []}
 
 A project initiated by [Angus Maddison](https://en.wikipedia.org/wiki/Angus_Maddison) has collected many historical time series that study economic growth. 
 
 We can use the [Maddison Historical Statistics](https://www.rug.nl/ggdc/historicaldevelopment/maddison/) to look at many different countries, including some countries dating back to the first century. 
 
 ```{tip}
-The data can be downloaded from [this webpage](https://www.rug.nl/ggdc/historicaldevelopment/maddison/) and clicking on the `Latest Maddison Project Release`. In this lecture we use the [Maddison Project Database 2020](https://www.rug.nl/ggdc/historicaldevelopment/maddison/releases/maddison-project-database-2020) using the `Excel` Format. The code we use here assumes you have downloaded that file and will teach you how to use [pandas](https://pandas.pydata.org) to import that data into a DataFrame.
+The data can be downloaded from [this webpage](https://www.rug.nl/ggdc/historicaldevelopment/maddison/) and clicking on the `Latest Maddison Project Release`. In this lecture we use the [Maddison Project Database 2020](https://www.rug.nl/ggdc/historicaldevelopment/maddison/releases/maddison-project-database-2020) using the `Excel` Format.
```

Comment:
"In this lecture" -> "Here" 
"using the" -> "in"
"Format" -> format.
"and clicking" -> "by clicking"